.travis.yml

```diff
@@ -2,6 +2,9 @@ language: generic
 services: docker
 
 script:
+  - >
+    docker run -v=$(pwd):/app -v=/tmp/coala:/cache
```

Comment:
why is `-v=/tmp/coala:/cache` here?

Reply:
@aashutoshrathi @jayvdb I'm wondering too :) It's not necessary

Reply:
No need here, fixing it.